tests/autoyast/installation.pm

```diff
@@ -132,6 +133,11 @@ sub run {
       = qw(bios-boot nonexisting-package reboot-after-installation linuxrc-install-fail scc-invalid-url warning-pop-up inst-betawarning autoyast-boot);
     push @needles, 'autoyast-confirm'        if get_var('AUTOYAST_CONFIRM');
     push @needles, 'autoyast-postpartscript' if get_var('USRSCR_DIALOG');
+    # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
+    # So push a needle to check upcoming reboot on zVM that is a way to indicate the stage done
+    push @needles, 'autoyast-stage1-reboot-upcoming' if check_var('BACKEND', 's390x');
```

Comment:
I still  wonder if we can do anything better than this needle, for example checking serial output. But let's try it out and see how we can improve.